pkg/localize/locales/en/cmd/kafka_delete.en.toml

```diff
@@ -27,11 +27,11 @@ $ rhoas kafka delete --id=1iSY6RQ3JKI8Q0OTmjQFd3ocFRg
 
 [kafka.delete.flag.id]
 description = 'Description for the --id flag'
-one = 'Unique ID of the Kafka instance you want to delete. If not set, the current Kafka instance will be used.'
+one = 'If unique ID not set for the Kafka instance you want to delete, the current Kafka instance will be utilised'
```

Comment:
We should probably rephrase it as `If a unique ID is not set for ...`. Wdyt?

Reply:
The existing sentence consists of a description followed by an instruction/disclaimer, the proposed description sounds bit more of an instruction and less of a description.  Wdyt?

Reply:
yep, done some more changes, please have a look.